django/contrib/contenttypes/migrations/0002_remove_content_type_name.py

```diff
@@ -33,6 +33,7 @@ class Migration(migrations.Migration):
         migrations.RunPython(
             migrations.RunPython.noop,
             add_legacy_name,
+            hints={'model_name': 'contenttype'},
```

Comment:
IMO people should try to stick to a single operation per model as much as possible. If I used multiple models I would probably do `{'model_names': ['model1', 'model2']}` but then the router has to play along, hardly something we can put in the docs.